src/utils/filterContests.test.js

```diff
@@ -0,0 +1,18 @@
+import { filterContests } from './filterContests';
+
+function testFilter(contests, from, to) {
+  let filteredContests = contests.filter((contest) => {
```

Comment:
Although this is pretty much guaranteed to work, the expected result should be hard coded.